tools/build

```diff
@@ -76,30 +79,70 @@ mkdir gcc_compile
 
 # Compile binutils
 cd binutils-$BINUTILS_V
-./configure --prefix=${INSTALL_PATH} --target=mips64-elf --with-cpu=mips64vr4300 --disable-werror
+./configure \
+  --prefix=${INSTALL_PATH} \
+  --target=mips64-elf \
+  --with-cpu=mips64vr4300 \
+  --disable-werror
 make -j$JOBS
 make install || sudo make install || su -c "make install"
 
-# Compile gcc (pass 1)
+# Compile GCC for MIPS N64 (pass 1)
 cd ../gcc_compile
-../gcc-$GCC_V/configure --prefix=${INSTALL_PATH} --target=mips64-elf --with-arch=vr4300 --with-tune=vr4300 --enable-languages=c --without-headers --with-newlib --disable-libssp --enable-multilib --disable-shared --with-gcc --disable-threads --disable-win32-registry --disable-nls --disable-werror --with-system-zlib
+../gcc-$GCC_V/configure \
+  --prefix=${INSTALL_PATH} \
+  --target=mips64-elf \
+  --with-arch=vr4300 \
+  --with-tune=vr4300 \
+  --enable-languages=c \ # c++ language support is added in the second pass.
```

Comment:
Unfortunately this doesn't work with bash. The continuation character `\` must be the last character on a line. You can't add the comment afterwards (you cannot even a space after it).

Reply:
Oops, I forgot about that. I will remove it. BTW, something in the back of my head tells me that C++ should also be enabled in the first pass, but I cannot remember why... But I guess it is easier to work out with the parameters on seperate lines if it ever becomes an issue.

Reply:
e.g. Would it fix: 
![image](https://user-images.githubusercontent.com/11439699/131569147-60c1a44a-def2-41c8-90f8-bffde25e1530.png)

Reply:
`__assert_func` is defined in `system.c` which is in `libdragonsys.a`. I see that it's being linked. So I don't know why the linker complains. That test compiles correctly for me, FWIW.

Reply:
I have just merged in the latest changes from your `trunk`. The issue has resolved itself.